.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "examples/FSharp.Data"]
+	path = examples/FSharp.Data
+	url = git@github.com:fsprojects/FSharp.Data.git
+[submodule "examples/Giraffe"]
+	path = examples/Giraffe
+	url = git@github.com:giraffe-fsharp/Giraffe.git
```

Comment:
I would suggest using F# compiler, specifically files in tests, since we add new syntax constructs the moment they are added.